INWX: Test PTR:Create_PTR_record failing

Currently it looks like one test with INWX is failing:

```
PS C:\dev\dnscontrol-src\integrationTest> go test -v -verbose -provider INWX -diff2
=== RUN   TestDNSProviders
=== RUN   TestDNSProviders/dnscontrol-inwx.com
[...]
--- FAIL: TestDNSProviders (151.26s)
    --- FAIL: TestDNSProviders/dnscontrol-inwx.com (150.96s)
[...]
        --- PASS: TestDNSProviders/dnscontrol-inwx.com/Post_cleanup:Empty#15 (0.54s)
        --- FAIL: TestDNSProviders/dnscontrol-inwx.com/18:PTR:Create_PTR_record (0.81s)
        --- PASS: TestDNSProviders/dnscontrol-inwx.com/Post_cleanup:Empty#16 (0.52s)
[...]
--- PASS: TestDualProviders (1.57s)
FAIL
exit status 1
FAIL    github.com/StackExchange/dnscontrol/v3/integrationTest  153.282s
```

Yeah, I was planning to. Interestingly:

```
    integration_test.go:223: DELETE A rec0088.domain.tld 1.2.3.4 ttl=300
    integration_test.go:223: DELETE A rec0089.domain.tld 1.2.3.4 ttl=300
    integration_test.go:223: DELETE A rec0090.domain.tld 1.2.3.4 ttl=300
    integration_test.go:223: DELETE A rec0091.domain.tld 1.2.3.4 ttl=300
    integration_test.go:223: DELETE A rec0092.domain.tld 1.2.3.4 ttl=300
panic: test timed out after 10m0s

goroutine 3442 [running]:
testing.(*M).startAlarm.func1()
        /usr/lib/go-1.19/src/testing/testing.go:2036 +0x8e
created by time.goFunc
```

Apparently the production API is way slower than the sandbox and the test was timing out.

But the test even fails with the production instance:
```
$ go test -v -verbose -timeout 0 -provider INWX -start 19 -end 19
=== RUN   TestDNSProviders
=== RUN   TestDNSProviders/domain.tld
=== RUN   TestDNSProviders/domain.tld/Clean_Slate:Empty
=== RUN   TestDNSProviders/domain.tld/19:PTR:Create_PTR_record
    integration_test.go:223: CREATE PTR 4.domain.tld foo.com. ttl=300
    integration_test.go:242: Expected 0 corrections on second run, but found 1.
    integration_test.go:244: UNEXPECTED #0: MODIFY PTR 4.domain.tld: (foo.com ttl=300) -> (foo.com. ttl=300)
=== RUN   TestDNSProviders/domain.tld/Post_cleanup:Empty
    integration_test.go:223: DELETE PTR 4.domain.tld foo.com ttl=300
--- FAIL: TestDNSProviders (8.15s)
    --- FAIL: TestDNSProviders/domain.tld (7.76s)
        --- PASS: TestDNSProviders/domain.tld/Clean_Slate:Empty (0.30s)
        --- FAIL: TestDNSProviders/domain.tld/19:PTR:Create_PTR_record (5.08s)
        --- PASS: TestDNSProviders/domain.tld/Post_cleanup:Empty (2.37s)
=== RUN   TestDualProviders
    integration_test.go:332: Clearing everything
    integration_test.go:339: Adding nameservers from another provider
    integration_test.go:326: #1: CREATE NS domain.tld ns1.example.com. ttl=300
    integration_test.go:326: #2: CREATE NS domain.tld ns2.example.com. ttl=300
    integration_test.go:342: Running again to ensure stability
--- PASS: TestDualProviders (4.54s)
FAIL
exit status 1
FAIL    github.com/StackExchange/dnscontrol/v3/integrationTest  12.699s
```

--- a/providers/inwx/inwxProvider.go
+++ b/providers/inwx/inwxProvider.go
@@ -308,7 +308,14 @@ func (api *inwxAPI) GetZoneRecords(domain string, meta map[string]string) (model
 		   Records with empty targets (i.e. records with target ".")
 		   are not allowed.
 		*/
-		if record.Type == "CNAME" || record.Type == "MX" || record.Type == "NS" || record.Type == "SRV" {
+		var rtypeAddDot = map[string]bool{
+			"CNAME": true,
+			"MX":    true,
+			"NS":    true,
+			"SRV":   true,
+			"PTR":   true,
+		}
+		if rtypeAddDot[record.Type] {
 			record.Content = record.Content + "."
 		}
 

